CHANGELOG.md

```diff
@@ -21,6 +22,7 @@
 ### Changed
 
 - `capture-stream` requires rustc version 1.49.0 due to dependency on `tokio`.
+>>>>>>> master
```

Comment:
Accidentally left over, please remove.

Reply:
@Wojtek242 , did you mean to remove line 25 only? Or to remove both line 24 and 25?